Midichlorians/MidichloriansDataDictionary.h

```diff
@@ -147,8 +145,41 @@ using T_IOS_EVENTS = TypeList<iOSFirstLaunchEvent,
 
 namespace web {
 
-}  // namespace web
+// Server collecting web events is supposed to be run behind nginx or other proxy.
```

Comment:
`Server` -> `The server`, and `Thus` is overcomplicating it. Some `Event collection server expects the following HTTP headers to be set:`  looks complete to me. We'll add an example nginx config into the Current repo and link to it from this comment, a `TODO(dkorolev)` about it would be helpful.